### PR TITLE
Use a dedicated terminal for each node

### PR DESCRIPTION
**Pull request by ignasi35**
References #635

@mckeeh3 how does this look?

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc [hunk 1]
```diff
@@ -109,20 +109,27 @@ mvn compile exec:exec -DAPP_CONFIG=local1.conf
...
 ----
 
-Similarly, you can start two additional nodes with `local2.conf` and `local3.conf`:
+Then, on separate terminal windows, you can start two additional nodes with `local2.conf` and `local3.conf`:
 
 [.group-java]
 [source,shell script]
 ----
-# make sure to compile before running exec:exec
+# On another terminal run:
```
Comment (thothmoses) [inline docs-source/docs/modules/microservices-tutorial/pages/template.adoc:117]: On a second terminal run:

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc [hunk 2]
```diff
@@ -109,20 +109,27 @@ mvn compile exec:exec -DAPP_CONFIG=local1.conf
...
+#   (make sure to compile before running exec:exec)
 mvn compile exec:exec -DAPP_CONFIG=local2.conf
+
+# And on a third terminal, run:
```
Comment (thothmoses) [inline docs-source/docs/modules/microservices-tutorial/pages/template.adoc:121]: On a third terminal run:

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc [hunk 3]
```diff
@@ -109,20 +109,27 @@ mvn compile exec:exec -DAPP_CONFIG=local1.conf
...
 mvn compile exec:exec -DAPP_CONFIG=local3.conf
 ----
 
 [.group-scala]
 [source,shell script]
 ----
+# On another terminal run:
```
Comment (thothmoses) [inline docs-source/docs/modules/microservices-tutorial/pages/template.adoc:128]: On a second terminal run:

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc (hunk 2 above)
Comment (thothmoses) [inline docs-source/docs/modules/microservices-tutorial/pages/template.adoc:121]: On a third terminal run:

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc [hunk 4]
```diff
@@ -109,20 +109,27 @@ mvn compile exec:exec -DAPP_CONFIG=local1.conf
...
 sbt -Dconfig.resource=local2.conf run
+
+
+# And on a third terminal, run:
```
Comment (thothmoses) [inline docs-source/docs/modules/microservices-tutorial/pages/template.adoc:132]: On a third terminal run:

**Review by thothmoses — COMMENTED** [review]
Looks good - My suggestion here is to use parallel language when enumerating.

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc (hunk 1 above)
Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/template.adoc:117]:
```suggestion
# On a second terminal run:
```

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc (hunk 2 above)
Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/template.adoc:121]:
```suggestion
# On a third terminal run:
```

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc (hunk 4 above)
Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/template.adoc:132]:
```suggestion
# On a third terminal run:
```

File: docs-source/docs/modules/microservices-tutorial/pages/template.adoc (hunk 3 above)
Comment (ignasi35) [reply docs-source/docs/modules/microservices-tutorial/pages/template.adoc:128]:
```suggestion
# On a second terminal run:
```

**Review by ignasi35 — COMMENTED** [review]
self-review to edit from GH UI

**Comment by ignasi35** [comment]
@thothmoses for small typos and oneliner improvements you can use [the `suggestion` feature](https://haacked.com/archive/2019/06/03/suggested-changes/) on GitHub's UI.

**Pull request merged**